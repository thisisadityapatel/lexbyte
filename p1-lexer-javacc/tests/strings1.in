""
"Test alphabet1: abcdefghijklmnopqrstuvwxyz"
"Test alphabet2: ABCDEFGHIJKLMNOPQRSTUVWXYZ"
"Test digits: 0123456789"
"Test other chars: `~!@#$%^&**() -_=+[{]}|;:',<.>/?"
"Multiple strings on the same line:" "string1""string2"
