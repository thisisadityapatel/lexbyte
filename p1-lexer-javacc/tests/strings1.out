String scanned: [""]
	->Token kind: <STRING>
	->Token value: ""
String scanned: ["Test alphabet1: abcdefghijklmnopqrstuvwxyz"]
	->Token kind: <STRING>
	->Token value: "Test alphabet1: abcdefghijklmnopqrstuvwxyz"
String scanned: ["Test alphabet2: ABCDEFGHIJKLMNOPQRSTUVWXYZ"]
	->Token kind: <STRING>
	->Token value: "Test alphabet2: ABCDEFGHIJKLMNOPQRSTUVWXYZ"
String scanned: ["Test digits: 0123456789"]
	->Token kind: <STRING>
	->Token value: "Test digits: 0123456789"
String scanned: ["Test other chars: `~!@#$%^&**() -_=+[{]}|;:',<.>/?"]
	->Token kind: <STRING>
	->Token value: "Test other chars: `~!@#$%^&**() -_=+[{]}|;:',<.>/?"
String scanned: ["Multiple strings on the same line:"]
	->Token kind: <STRING>
	->Token value: "Multiple strings on the same line:"
String scanned: ["string1"]
	->Token kind: <STRING>
	->Token value: "string1"
String scanned: ["string2"]
	->Token kind: <STRING>
	->Token value: "string2"
